frontend/src/entity/TeamCard/ui/TeamCardStatistics/TeamCardStatistics.style.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Avatar,
+  Box,
+  TableBody,
+  TableCell,
+  TableHead,
+  Typography,
+} from "@mui/material";
+import styled from "styled-components";
+import { theme } from "../../../../app";
+
+export const IconBlock = styled(Box)(() => ({
+  display: "flex",
+  gap: "16px",
+}));
+
+export const SecondaryText = styled(Typography)(() => ({
+  "&&": {
+    color: theme.palette.text.secondary,
+    fontSize: "14px",
+  },
+}));
+
+export const MainText = styled(Typography)(() => ({
+  "&&": {
+    fontSize: "16px",
+    lineHeight: "24px",
+  },
+}));
+
+export const LeagueBlock = styled(Box)(() => ({
+  display: "flex",
+  fontSize: "13px",
+  backgroundColor: theme.palette.customColors.main,
+  borderRadius: "16px",
+  padding: "0 8px",
+  height: "fit-content",
+  width: "fit-content",
+}));
+
+export const CardWrapper = styled(Box)(() => ({
+  padding: "16px",
+  backgroundColor: theme.palette.background.paper,
+  width: "536px",
+  marginTop: "16px",
+  display: "flex",
+  flexDirection: "column",
+  border: "1px solid #E0E0E0",
```

Comment:
import { grey } from '@mui/material/colors'
.... 
```suggestion
  border: `1px solid ${grey[300]}`,
```